webhooks-extension/src/WebhookApp.js

```diff
@@ -105,7 +107,9 @@ const mapDispatchToProps = (dispatch, props) => ({
   fetchPipelines: namespace =>
     dispatch(props.actions.fetchPipelines({ namespace })),
   fetchServiceAccounts: namespace =>
-    dispatch(props.actions.fetchServiceAccounts({ namespace }))
+    dispatch(props.actions.fetchServiceAccounts({ namespace })),
+  fetchPipelineRuns: (namespace, pipelineName, filters) =>
+    dispatch(props.actions.fetchPipelineRuns(namespace, pipelineName, filters))
```

Comment:
the `fetchPipelineRuns` action expects an object as param

Reply:
i thought this was the way to go but for some reason when I surround the variables with `{}`, in the `PipelineRun` action it initializes namespace with the object passed and filters variable ends up being undefined....